src/App.css

```diff
@@ -18,19 +18,19 @@ nav a {
   display: flex;
   flex-direction: column;
   justify-content: center;
+  align-items: center;
   text-decoration: none;
   color: black;
-  font-size: 1.1rem;
-  padding: 3% 0;
+  font-size: 1rem;
+  padding: 2% 0;
   margin: 2%;
   width: 30%;
-  border: 3px solid black;
+  border-radius: 5px;
 }
 
```

Comment:
I altered the css here just a little to get the style a little closer to our wireframe. I am realizing I should move this styling into the Navigation.css though, so will do that and push those changes to this branch.